CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(drivers)
```

Comment:
A comment explaining that this is picked up by the zephyr build system and is required to make the code in drivers/ visible to that build system would be good.